bindings/pydeck/pydeck/bindings/deck.py

```diff
@@ -18,15 +17,14 @@ def __init__(
         layers=[],
         views=[View(type="MapView", controller=True)],
         map_style=DARK,
-        mapbox_key=None,
-        google_maps_key=None,
+        api_keys={},
```

Comment:
Nit: Avoid mutable default arguments ([1](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments), [2](https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument)). This has bitten me recently. (applies to the `list` arguments above as well`)

Reply:
Interesting point--in over a decade of writing Python I've never thought about this.